src/components/Category.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './Styles/Category.css';
+
+const Category = () => (
+  <button type="submit" className="statusbtn"> Status</button>
```

Comment:
- [ ] [OPTIONAL] I would highly recommend changing the name of the button to `Check status` it's more descriptive and contain a call to action, this is better for UX